### DISCUSSION TITLE
Closed mini HUD on screen.

### DISCUSSION THREAD

**Discussion by Joshfrom307** [Courseplay/Courseplay_FS22#2290] [Q&A] [answered]
![20230206_155121](https://user-images.githubusercontent.com/124636739/217106259-b84e50f9-f1b7-41c4-9563-3bbd23c53da8.jpg)
New to CP and was messing with the settings . CP version is 7.2.0.0 - FS22 and I'm trying to get this box in the pic to come back .I've tried changing keybindings ......is this a bug ?

**Comment by Tensuko** [comment, marked as answer]
No, it is the info panel.
Read ingame help for more details.
